src/frontend/src/helpers/Storage/LocalStorage.ts

```diff
@@ -91,12 +52,6 @@ class Store implements StorageInterface {
   // Will be replaced by Dexie.Syncable.ISyncProtocol
   public async getProjectsForSync(): Promise<Project[]> {
     return this.db.projects.toArray();
```

Comment:
Yeah these are simple fixes but we _need_ to integrate them before we deploy as otherwise problems.